pkg/apis/serving/metadata_validation.go

```diff
@@ -63,6 +63,20 @@ func validateKnativeAnnotations(annotations map[string]string) (errs *apis.Field
 	return
 }
 
+const noAutoscalingMsg = "annotations in group 'autoscaling.knative.dev' won't work here, they must be put under 'spec.template.metadata.annotations' instead"
```

Comment:
```suggestion
const noAutoscalingMsg = `annotations in group "autoscaling.knative.dev" won't work here, they must be put under "spec.template.metadata.annotations" instead`
```
I think a more mainstream formatting, but feel free to ignore :)